Update projects.mdx (#114)
launchdarkly/LaunchDarkly-Docs: src/content/topics/home/managing-flags/projects.mdx

Set up a project to serve one set of end users. Sorting your flags into related chunks of functionality based on their impact on end users helps you logically organize related flags.

> A common pattern is to map each project in your LaunchDarkly instance to a specific product your company makes. For example, a mobile application, web app, and backend services that are presented to the user as one product might all fit in one project since they likely will share most flags. Other common patterns on how to map your architecture with LaunchDarkly, read [Implementing LaunchDarkly Project Architecture Best Practices](https://launchdarkly.com/blog/implementing-launchdarkly-project-architecture-best-practices/).

Reviewer: ```suggestion
A common pattern is to map each project in your LaunchDarkly instance to a specific product your company makes. For example, a mobile application, web app, and backend services that are presented to the user as one product might all fit in one project since they likely will share most flags. To learn about other common patterns for mapping your architecture with LaunchDarkly, read [Implementing LaunchDarkly Project Architecture Best Practices](https://launchdarkly.com/blog/implementing-launchdarkly-project-architecture-best-practices/).
```
